src/components/layout-panels/leftPanel/LeftPanel.tsx

```diff
@@ -23,6 +25,7 @@ function LeftPanel({ variables, isDocumentLoaded }: LeftPanelProps) {
             panelTheme={panel}
         >
             <VariablesListContainer hidden={contentType === ContentType.IMAGE_PANEL}>
+                {isDocumentLoaded && isSandboxMode ? <DataSource /> : null}
```

Comment:
It should be available in the regular mode as well. Please just hide it under FF `STUDIO_DATA_SOURCE` for now